src/gsp.py

```diff
@@ -286,6 +313,10 @@ def get_truths_for_a_specific_gsp_old_route(
     user: Auth0User = Security(get_user()),
 ) -> List[GSPYield]:
     """Redirects old API route to new route /v0/solar/GB/gsp/{gsp_id}/pvlive"""
+
+    if is_fake:
+        make_fake_forecast(gsp_id=gsp_id, session=session)
```

Comment:
This is where we should use `make_fake_gsp_yields`, and not `make_fake_forecast`

Reply:
Ah, right. That makes more sense! I've made and pushed the changes now. I might need to run a few unsuccessful tests before getting it right :).